### PR TITLE
Editor shortcuts

### LINKED ISSUE #1104: Editor shortcuts
keyboard shortcuts for editor (Add image / bold / tables / Add link / blockquote / code block / ....)

### PR DESCRIPTION
**Pull request by MTSaeed**
**What does this PR?**

Create shortcuts of keyboard while creating the post.
Shortcuts for different functionalities are given below:
Bold = ctrl + b
Italic = ctrl + i
To make new table = alt + t
To create a link = alt + k
For code block = alt + c
To upload an image = alt +d
To add blockquote = alt + m
 
**Steps to reproduce**

1. Open the create post page.
2. white some text and apply the shortcuts.

**Issue number** 

Fixes #1104

**Screenshot/Video**

https://user-images.githubusercontent.com/106739598/206391200-d6c5a818-77b8-4f9c-b3eb-5bff22f27f34.mp4

### CODE REVIEW

**Review by feruzm — CHANGES_REQUESTED** [review]
Can we stick with either Ctrl or Alt combination? I would prefer Alt, also on hover on those icons in editor toolbar, we should show tooltip about these shortcuts, so people can learn/see there is shortcut option.

**Comment by MTSaeed** [comment]
> Can we stick with either Ctrl or Alt combination? I would prefer Alt, also on hover on those icons in editor toolbar, we should show tooltip about these shortcuts, so people can learn/see there is shortcut option.

We could use alt in all. But it is more intuitive to use ctrl + b and ctrl + i for bold and italic as these shortcuts have become vast practices. What is your opinion about this?

**Comment by feruzm** [comment]
@MTSaeed We shouldn't forget OS and Browser shortcuts, people might want to keep those keys. Let's do Alt and gather community feedback, if we get more requests to change it to Ctrl, we can do so easily.

File: src/common/pages/submit.tsx [hunk 1]
```diff
@@ -847,6 +848,30 @@ class SubmitPage extends BaseComponent<Props, State> {
     return makeJsonMetaData(meta, tags, summary, version);
   };
 
+  shortcuts = (e: any) => {
```
[the remark this answers is not in the document]
Comment (dkildar) [reply src/common/pages/submit.tsx:851]: Second one: functions names should describe an action. In this case: `handleShortcuts`

[the remark this answers is not in the document]
Comment (MTSaeed) [reply src/common/pages/submit.tsx]: Updated.

**Review by feruzm — APPROVED** [review]
lgtm

**Pull request merged**